bigtable/tests/system.py

```diff
@@ -356,6 +356,32 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         cell4 = Cell(CELL_VAL4, timestamp4)
         return cell1, cell2, cell3, cell4
 
+    def test_mutate_rows(self):
+        row_1 = self._table.row(ROW_KEY)
+        row_1.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL1)
+        row_1.commit()
+        row_2 = self._table.row(ROW_KEY_ALT)
+        row_2.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL2)
+        row_2.commit()
+        rows = [row_1, row_2]
+        self.rows_to_delete.extend(rows)
+
+        # Change the contents
+        row_1.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL3)
+        row_2.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL4)
+        statuses = self._table.mutate_rows(rows)
+        result = [status.code for status in statuses]
+        expected_result = [0, 0]
+        self.assertEqual(result, expected_result)
+
+        # Check the contents
+        row_1_data = self._table.read_row(ROW_KEY)
+        self.assertEqual(
+            row_1_data.cells[COLUMN_FAMILY_ID1][COL_NAME1][0].value, CELL_VAL3)
```

Comment:
Between `getattr` and `__getitem__`, there are 5 layers deep of accesses in:

```
row_1_data.cells[COLUMN_FAMILY_ID1][COL_NAME1][0].value
```

Can we not just construct what `cells` should be a do a full comparison?